CONTRIBUTING.markdown

```diff
@@ -1,114 +1,183 @@
-you are using Liferay IDE, use the built-in code formatter accessible via the
-*Java* &rarr; *Code Style* &rarr; *Formatter* &rarr; *Active Profile* menu.
-* Include a reference to your ticket (e.g. LPS-XXXXX) in your commit messages.
-For example:
-
-        LPS-83432 Make the example in CONTRIBUTING imperative and concrete
-
-* *Test* your changes thoroughly! Consider the wide variety of operating
-systems, databases, application servers, and other related technologies Liferay
-supports. Make sure your changes in one environment don't break something in
-another environment. See [Unit and Integration
-Tests](http://www.liferay.com/community/wiki/-/wiki/Main/Unit+and+Integration+tests)
-for details on executing Liferay's automated tests.
-
-## Submitting Changes
-
-* Push changes in your branch to your fork.
-* Send a pull request to the core maintainer of the area to which your changes
-apply. You can use [github.com](https://github.com/) to initiate the pull
-request, or you can use
-[git-pull-request](https://github.com/liferay/git-tools/tree/master/git-pull-request),
-a command line tool for issuing pull requests, developed specifically for the
-Liferay Community.
-* In the LPS ticket, provide a link to your GitHub pull request and respond to
-the [Contributor License
-Agreement](http://www.liferay.com/legal/contributors-agreement) displayed by
-clicking the *Contribute Solution* button.
-* You're done! Well, not quite ... be sure to respond to comments and questions
-to your pull request until it is closed.
-
-## Additional Resources
-
-* [Getting Started as a Liferay
-Developer](http://www.liferay.com/community/wiki/-/wiki/Main/Getting+started+as+a+Liferay+Developer+in+a+few+steps)
-* [Optimal Liferay Core
-Development](http://www.liferay.com/community/wiki/-/wiki/Main/Optimal+Liferay+Core+Development)
-* [Liferay and JIRA](http://www.liferay.com/community/wiki/-/wiki/Main/JIRA)
-* [Contribute to Liferay on
-Github](http://www.liferay.com/community/wiki/-/wiki/Main/Contribute+using+Git+and+GitHub)
-* [Liferay Core Development
-Guidelines](http://www.liferay.com/community/wiki/-/wiki/Main/Liferay+Core+Development+Guidelines)
-* [Setting up and using Liferay
-IDE](http://www.liferay.com/community/wiki/-/wiki/Main/Liferay+Contributor+Development+Environment+Setup)
-* [Contributor License
-Agreement](http://www.liferay.com/legal/contributors-agreement)
-* [General GitHub documentation](http://help.github.com/)
-* [GitHub pull request
-documentation](http://help.github.com/send-pull-requests/)
-* [Liferay's IRC channel on
-freenode.org](http://webchat.freenode.net/?channels=liferay&uio=d4)
\ No newline at end of file
+
+# Liferay Contribution Process
+
+Liferay 7 went from being a single monolithic application to being composed of many modules. This has lead to changes in the way that we develop for Liferay as well as how we accept contributions from the community.
```

Comment:
I think it's better to start with something about how much we appreciate contributions, how much it's helped shape the product into what it is today and how many benefits there are to contributing.

I don't think talking about how things have changed is so important here because it will only be relevant to those who have been around the longest time but not to new comers who will be the ones who will probably look at this file anyway.